### DISCUSSION TITLE
Open/arrange windows on startup when program launches multiple windows

### DISCUSSION THREAD

**Discussion by aaannestad** [i3/i3#6099] [Q&A]
I'm interested in taking advantage of i3's ability to automatically open and arrange programs on startup, but I've got a situation I'm not sure how to handle best. I use my computer to drive my TV, and usually watch stuff on my TV via a browser window; I also have a separate browser window for my main monitor, and I allow the browser to remember the session as a whole and simply recreate both windows again when launched. How can I specifically place these separate windows in the right workspaces, when both are opened via a single command to trigger the browser?

**Comment by orestisfl** [comment]
Take a look at https://i3wm.org/docs/layout-saving.html

**Reply by aaannestad** [reply]
I see vague hints of things that might be answers in there, but no clear answer (^^) Is the 'instance' property what I'm looking for? Is there a way to see an instance name for something I haven't set myself?
